### PR TITLE
chore(dependency)

### PR DESCRIPTION
**Pull request by tobiaskneubuehler**
update dependencies (mobx still on 4.x), provisions for mobx 5

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -60,7 +61,10 @@ module.exports = [
             extensions: ['.js', '.jsx']
         }
         , plugins: [
-            new TapWebpackPlugin({ reporter: 'node ./node_modules/tap-spec/bin/cmd' })
+            new TapWebpackPlugin({ reporter: 'node ./node_modules/tap-spec/bin/cmd' }),
+            // node-fetch uses the «encoding» package, but doesn't have it in it's dependencies.
```
[the remark this answers is not in the document]
Comment (fxstr) [reply webpack.config.js]: Oh: and multiline-comment. And it's 'its', not 'it's' ;-)

File: src/models/resistances/resistance.js [hunk 2]
```diff
@@ -43,7 +43,18 @@ export default class Resistance {
...
-        return this.values.sort((a, b) => (a.type.precision > b.type.precision ? -1 : 1));
+        /**
+        * As «this.values» is an observableArray:
+        *
+        * Unlike the built-in implementation of the functions sort and reverse, observableArray.sort
+        * and reverse will not change the array in-place, but only will return a sorted/reversed copy.
+        * From MobX 5 and higher this will show a warning.
+        * It is recommended to use array.slice().sort() instead.
+        * 
+        * observableArray.replace(observableArray.slice().sort()) to sort & update in place
```
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply src/models/resistances/resistance.js]: It's not old code, it's an example if someone wants to sort and update an array in place. I think it's ok to remove the whole comment right? Or do we need this information?

File: webpack.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply webpack.config.js]: :+1:

[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply webpack.config.js]: :+1:

File: src/models/resistances/resistance.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (fxstr) [reply src/models/resistances/resistance.js]: I share your opinion. It's common MobX 5 knowledge, we can remove the comment.

File: src/helpers/store.unit.js [hunk 3]
```diff
@@ -111,13 +111,19 @@ test('setting fetchPromise updates status and promise', (t) => {
 test('sets status to error on errors', (t) => {
 	const store = new Store();
 	store.setFetchPromise(new Promise((resolve, reject) => {
-		setTimeout(() => {
-			reject();
-		}, 50);
+		setTimeout(reject, 50);
```
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply src/helpers/store.unit.js]: ;)

File: src/models/resistances/resistance.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply src/models/resistances/resistance.js]: :+1:

File: src/helpers/store.unit.js [hunk 4]
```diff
@@ -111,13 +111,19 @@ test('setting fetchPromise updates status and promise', (t) => {
...
 	}));
+
 	observe(store.status, (status) => {
-		t.equals(store.status.identifier, 'error');
-		t.end();
+		// we do not want to track the status type 'add'
```
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply src/helpers/store.unit.js]: Added some lines of comments so it's clearer. The problem is, if we track both «add» and «update» the reaction is fired twice, so is the t.end() method and the test fails

**Comment by tobiaskneubuehler** [comment]
@fxstr fixed as commented

**Pull request merged**